src/tools/sss_override.c

```diff
@@ -333,13 +333,21 @@ static struct sysdb_attrs *build_attrs(TALLOC_CTX *mem_ctx,
         if (ret != EOK) {
             goto done;
         }
+    }else {
```

Comment:
Hello Fabiano,
Greatly appreciate your comments. Will take care in future commits. Sincere Apologies for this time.
Extra space add: Done